### PR TITLE
Feat/es6

### PR DESCRIPTION
**Pull request by GitAndrew**
- Added babel for es6 support
- Refactored code to use es6 imports instead of commonjs require
- Added nodemon for hot loading (for dev)
- Updated package.json scripts - serve for prod, start for dev

Couple of issues I encountered along the way:
- Needed to add --copy-files to babel cli call (see within npm run serve) because by default babel will not compile/copy over the .graphql files into the dist folder
- Encountered an issue with babel compiling bunyan logger, needed to add the plugin: @babel/plugin-transform-runtime
https://stackoverflow.com/questions/61755999/uncaught-referenceerror-regeneratorruntime-is-not-defined-in-react

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+      "@babel/preset-env"
+    ],
+    "plugins": ["@babel/plugin-transform-runtime"]
```
Comment (GitAndrew) [inline .babelrc:5]: As mentioned, needed this to for babel to compile the loggers. Remove and see for yourself when running 'npm run serve'
https://stackoverflow.com/questions/61755999/uncaught-referenceerror-regeneratorruntime-is-not-defined-in-react

File: .eslintrc.js [hunk 2]
```diff
@@ -4,12 +4,13 @@ module.exports = {
     es2021: true,
     node: true,
   },
-  extends: ["prettier", "airbnb-base"],
+  extends: ["airbnb-base", "prettier"],
```
Comment (GitAndrew) [inline .eslintrc.js:7]: Swapping these two resolved the conflicts I had for auto-formatting on save

File: jsconfig.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src"
+  },
+  "include": ["src"]
+}
```
Comment (GitAndrew) [inline jsconfig.json:6]: Added this file so you can do relative imports to src in vscode

File: package.json [hunk 4]
```diff
@@ -7,7 +7,11 @@
     "node": "^16.3.0"
   },
   "scripts": {
-    "start": "node bin/www",
+    "clean": "rm -rf dist",
+    "transpile": "babel src -d dist --copy-files",
+    "build": "npm run clean && npm run transpile",
+    "serve": "npm run build && node dist/bin/www.js",
+    "start": "nodemon --exec babel-node src/bin/www.js",
```
Comment (GitAndrew) [inline package.json]: For prod: npm run serve. Compiles/nuilds into a dist folder and runs the bin/www.js script. Note the --copy-files as mentioned in the PR description.
For dev: npm run start. Runs nodemon for hot loading + compiles with babel-node

babel-node is not appropriate for production: https://babeljs.io/docs/en/babel-node.html

File: src/bin/www.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import dotenv from "dotenv";
+import server from "..";
+
+dotenv.config();
+
+server.listen().then(({ url }) => {
```
Comment (chrisrabe) [inline src/bin/www.js:6]: hmm might need to change the main script to src/bin/www in package.json

**Review by chrisrabe — CHANGES_REQUESTED** [review]
Need to change the location of the main script in package.json

**Review by chrisrabe — APPROVED** [review]
looks good

**Pull request merged**